server/models/Message.js

```diff
@@ -0,0 +1,9 @@
+let mongoose = require("mongoose");
+
+const Message = new mongoose.Schema({
+    author: mongoose.ObjectId,
+    body: String,
+    time: { type: Date, default: Date.now },
+});
+
+module.exports = mongoose.model("message", Message);
```

Comment:
```suggestion
export { Message }
```
But don't import as a model, only import Schema. Check out [this](https://mongoosejs.com/docs/subdocs.html)